packages/material-ui/src/Typography/Typography.d.ts

```diff
@@ -117,6 +119,10 @@ export interface TypographyTypeMap<P = {}, D extends React.ElementType = 'span'>
      * @default false
      */
     paragraph?: boolean;
+    /**
+     * The system prop that allows defining system overrides as well as additional CSS styles.
+     */
+    sx?: SxProps<Theme>;
```

Comment:
@mnajdova Should we take this change back to the component in an isolate PR? It was forgotten to in the Avatar component. 

On a different note, would it be simpler to do:

```diff
-import { SxProps } from '@material-ui/system';
-import { Theme } from '@material-ui/core/styles';
+import { BoxProps } from '@material-ui/core/Box';

-sx?: SxProps<Theme>;
+sx?: BoxProps['sx'];
```

Reply:
Oops, yes, will take a look tomorrow, I believe it is not added on the Buttons too. I don’t have a preference on which type we should use, as long as we add it :))